cmd/provenanced/cmd/docgen.go

```diff
@@ -0,0 +1,103 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/cobra/doc"
+
+	"github.com/cosmos/cosmos-sdk/version"
+)
+
+var docGenCmdStart = fmt.Sprintf("%s docgen", version.AppName)
+
+const (
+	FlagMarkdown = "markdown"
+	FlagYaml     = "yaml"
+	FlagRest     = "rest"
```

Comment:
The "rest" flag here was unexpected vs "rst" given how strongly  "rest" is associated with http services in my mind.

Reply:
I agree. I updated it to "rst".